.license-tools-config.json

```diff
@@ -0,0 +1,40 @@
+{
+  "author": {
+    "name": "Lukas Burgholzer",
+    "years": [2025]
```

Comment:
We should update this before finalising.
Either your name, @denialhaag, should appear here too, or we go with MQSC or MQSC+TUM.